core/src/test/scala/com.mediative.sparrow/DataFrameReaderTest.scala

```diff
@@ -168,58 +153,12 @@ class DataFrameReaderTest extends FreeSpec with BeforeAndAfterAll {
     }
 
     def testFailure[T: RowConverter: ClassTag](json: Array[String], expected: NonEmptyList[String]) = {
+      val sqlContext = new SQLContext(sc)
       val df = sqlContext.jsonRDD(sc.parallelize(json))
 
       assert(toRDD[T](df) == expected.failure)
     }
 
-    // To get DataFrame#toRDD usage.
-    import com.mediative.sparrow.syntax.df._
-
-    "round-trip an object containing an Int and a String from RDD to DataFrame back to RDD" in {
```

Comment:
@suhailshergill I'm removing two tests that don't personally think were because they overlapped with existing tests. The one that was pending was replaced by a more targeted test. 

What `createDataFrame` does is pretty straight forward and I don't personally see that much value of it rather than just manually create a `Row` the same way as they would be created by `createDataFrame`, which is what the existing tests already do.

If you want to keep them anyway I would strongly suggest having those in separate spec. But I leave this PR for you and @jonas to handle, I'm in vacation, I don't plan on updating this PR any further. I just thought I could easily fix issue that you said was that severe. So you have to fix to include if you need it and the pattern to follow if similar treatment should be applied to other cases.